<H2>Description</H2>
<P>
This track shows an estimate of RNA abundance (transcription) 
over the first  24 hours of <em>D. melanogaster</em> development in 
two hour increments, measured by a tiling array as described in 
Manak <em>et al</em>. (2006) (see References).
Composite signals are shown in
separate subtracks for each of the twelve timepoints.</P>

<H2>Display Conventions and Configuration</H2>
<P>
The subtracks within this composite annotation track
may be configured in a variety of ways to highlight different aspects of the 
displayed data. The graphical configuration options for the subtracks 
are shown at the top of the track description page, followed by a list of 
subtracks. To show only selected subtracks, uncheck the boxes next to 
the tracks that you wish to hide.  
For more information about the graphical configuration options, click the 
<A HREF="../goldenPath/help/hgWiggleTrackHelp.html" TARGET=_blank>Graph
configuration help</A> link.</P>
<P>
Color differences among the subtracks are arbitrary. They provide a
visual cue for distinguishing between the different timepoints.</P>

<H2>Methods</H2>
<P>
The data were processed into signal and transfrags as described in 
Cheng <em>et al</em>. (2005) and Kampa <em>et al</em>. (2004).  
The data from replicate arrays were quantile-normalized and all arrays
were scaled to a median array intensity of 25.  Within a sliding 101
bp window centered on each probe, an estimate of RNA abundance
(signal) was found by calculating the median of all pairwise average
PM-MM values, where PM is a perfect match and MM is a mismatch.  This data 
set was originally created for dm2(BDGP R4) and then lifted over to dm3(BDGP R5).
</P>

<H2>Verification</H2>
<P>
Samples were hybridized
to duplicate arrays (three technical replicates). Transcribed regions
were generated from the composite signal track by merging genomic positions
to which probes are mapped. This merging was based on a 5% false
positive rate cutoff in negative bacterial controls, a maximum
gap (MaxGap) of 50 base-pairs and minimum run (MinRun) of 90 base-pairs (see
the Affy TransFrags track for the merged regions).

</P>

<H2>Credits</H2>
<P>
These data were generated and analyzed by the Tom Gingeras group at 
<A HREF="http://www.affymetrix.com/" TARGET=_blank>Affymetrix</A>.</P>

<H2>References</H2>
<P>
Please see the 
<A HREF="http://transcriptome.affymetrix.com/local_index.html"
TARGET=_blank>Affymetrix Transcriptome</A> site for a project overview and
additional references to Affymetrix tiling array publications.</P>
<P>
Manak JR, Dike S, Sementchenko V, Kapranov P, Biemar F, Long J, Cheng J, 
Bell I, Ghosh S, Piccolboni A, Gingeras TR.
<A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16951679&query_hl=1&itool=pubmed_docsum" 
TARGET=_BLANK>Biological function of unannotated transcription during the 
early development of Drosophila melanogaster</A>. 
<em>Nature Genet</em>. 2006 Oct;38(10):1151-8.</P>
<P>
Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, Patel S, Long J, Stern D, Tammana H, Helt G, <em>et al</em>.
<A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&cmd=Retrieve&db=PubMed&list_uids=15790807&dopt=Abstract" 
TARGET=_BLANK>Transcriptional maps of 10 human chromosomes at 5-nucleotide 
resolution</A>. 
<em>Science</em>. 2005 May 20;308(5725):1149-54.</P>
<P>
Kampa D, Cheng J, Kapranov P, Yamanaka M, Brubaker S, Cawley S, Drenkow J, Piccolboni A, Bekiranov S, Helt G, <em>et al</em>.
<A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&cmd=Retrieve&db=PubMed&list_uids=14993201&dopt=Abstract" 
TARGET=_BLANK>Novel RNAs identified from an in-depth analysis of the 
transcriptome of human chromosomes 21 and 22</A>. 
<em>Genome Res</em>. 2004 Mar;14(3):331-42.</P>
